hc/front/tests/test_add_channel.py

```diff
@@ -26,16 +26,48 @@ def test_it_trims_whitespace(self):
         self.client.login(username="alice@example.org", password="password")
         self.client.post(url, form)
 
-        q = Channel.objects.filter(value="alice@example.org")
-        self.assertEqual(q.count(), 1)
+        channels = Channel.objects.filter(value="alice@example.org")
+        self.assertEqual(channels.count(), 1)
 
     def test_instructions_work(self):
         self.client.login(username="alice@example.org", password="password")
         kinds = ("email", "webhook", "pd", "pushover", "hipchat", "victorops")
         for frag in kinds:
             url = "/integrations/add_%s/" % frag
-            r = self.client.get(url)
-            self.assertContains(r, "Integration Settings", status_code=200)
+            response = self.client.get(url)
+            self.assertContains(
+                response,
+                "Integration Settings",
+                status_code=200)
 
-    ### Test that the team access works
-    ### Test that bad kinds don't work
+    def test_team_access_works(self):
```

Comment:
This test is still not testing the correct parameters. The file is called `test_add_channel`. The idea is that you use add a channel using alice's account and then login as bob and check to see that you can see the channel. Kindly reach out if this is not clear so that I can elaborate further in person